BnbnavNetClient/Services/TextToSpeech/ITextToSpeechProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BnbnavNetClient.Services.TextToSpeech;
+interface ITextToSpeechProvider
+{
+    Task SpeakAsync(string text, CultureInfo culture);
+}
```

Comment:
```suggestion
    public CultureInfo Culture { get; set; }
}
```

Make the culture a global thing